pkgs/by-name/bm/bmputil/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "bmputil";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "blackmagic-debug";
+    repo = pname;
```

`repo = pname;` is an antipattern as both names are arbitrary and are not necessarily always equal 
```suggestion
    repo = "bmputil";
```